plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiSessionProperties.java

```diff
@@ -200,4 +213,9 @@ public static boolean isIgnoreAbsentPartitions(ConnectorSession session)
     {
         return session.getProperty(IGNORE_ABSENT_PARTITIONS, Boolean.class);
     }
+
+    public static int getParquetMaxReadBlockRowCount(ConnectorSession session)
```

Comment:
Which place we are using it?
Shall we add it into `HudiPageSourceProvider`  cc @raunaqmorarka